### ISSUE TITLE
Unable to get stdout of the 'cat'ing of a file that contains (and starts with) special characters

### ISSUE THREAD

**Issue by inghama** [ansible/ansible#7299]
I am at version 1.5.5-1.el6 of ansible.

I issue this command:

ansible HOSTABC -a 'cat /etc/motd' -K --sudo -v
sudo password: 

```
 and I see the success message PLUS the stdout of that file
```

HOSTABC | success | rc=0 >>

You are about to access a Duke University computer ...

---

However, when I substitue a file that has special characters (and STARTS with one), I _do not see_ the stdout output, even though the command succeeds.

ansible HOSTABC -a 'cat /var/log/boot.log' -K --sudo -v
sudo password: 

HOSTABC | success | rc=0 >>

<STDOUT SHOULD BE *HERE*>

Can anyone shine a light on this for me?

Thanks!
Andy

**Comment by jimi-c** [comment]
I believe we fixed this issue in 1.6, if you can check against that version.

**Comment by bcoca** [comment]
closing as this seems to have been fixed long ago
